### PR TITLE
Update QUEST.tsv [Lines 1501-1650]

### PR DESCRIPTION
**Pull request by ghost**
Lines 1501-1650 reviewed. Open to feedback.

연판장 seems to have been translated as "compact" which makes no sense in the contexts it's used in. It machine-translates into "signature list", however, so I changed it to "petition." If that's wrong, feel free to correct.

Changed "contaminating gap" to "corruption breach." If anyone can think of a better keyword or think the original is fine then I'm okay with changing it to that or back to the original.

I'd like some input before this is merged.

### CODE REVIEW

File: QUEST.tsv [hunk 1]
```diff
@@ -1499,155 +1499,155 @@ QUEST_20150323_001498	Okay, that'll be only 1,000 silver coins then.
...
 QUEST_20150323_001528	Don't bother me.
-QUEST_20150323_001529	Then, I will sign the compact if you pay 10,000 silver.
-QUEST_20150323_001530	I haven't seen anyone as open-handed as you! So cooperative! Hahaha! Take this!
-QUEST_20150323_001531	What a penniless person. Go away.
-QUEST_20150323_001532	That's some perseverance. Well, I don't really trust you but I'll sign for Horacius' sake.
-QUEST_20150323_001533	Oh you… Hmm the compact of Horacius… Finally, the day of Rebellion is coming… Ok, I'll sign it.
-QUEST_20150323_001534	Did you use the tools well last time? A compact this time, sure I'll sign it. For victory.
-QUEST_20150323_001535	Let me read the details of the compact.
-QUEST_20150323_001536	Oh this is right. Here, let me sign.
+QUEST_20150323_001529	$Then, I will sign the petition if you pay 10,000 silver.
```
Comment (zhouyu47) [inline QUEST.tsv]: How about "contract"?

File: QUEST.tsv [hunk 2]
```diff
@@ -1499,155 +1499,155 @@ QUEST_20150323_001498	Okay, that'll be only 1,000 silver coins then.
...
 QUEST_20150323_001585	Device is operating
-QUEST_20150323_001586	A few addicted farmers assaulted the people. 
+QUEST_20150323_001586	$A few manic farmers assaulted the people. 
 QUEST_20150323_001587	Guards are working hard to protect the farm. {nl}Help them. 
-QUEST_20150323_001588	I thank you again in the name of all fate. {nl}From here, I will follow the stars to find Goddess Ausrine.{nl}
-QUEST_20150323_001589	The seal of Uskis Arable Land is now safe with your help. {nl}Other Revelators lost their consiousness due to the evil force coming from the Spring Light Woods.{nl}
-QUEST_20150323_001590	This Cathedral was built 660 years ago. {nl}It was built as the most glorious church under the guide of archbishop Silvestro Mayvern.{nl}
-QUEST_20150323_001591	It's like I can still hear it. {nl}Prayers of the priests filled the kingdom. {nl}There were endless march of Pilgrims coming for the grace of the Goddess.{nl}
-QUEST_20150323_001592	It was like that until four years ago. {nl}Before the curse surrounded the cathedral on Divine Day…
+QUEST_20150323_001588	$I thank you again in the name of fate. {nl}From here on, I will climb the stars to find Goddess Ausrine.
```
Comment (terrapin47) [inline QUEST.tsv]: "follow"ing the stars is something an astrologist would do, in order to discern fate.  So the original makes sense, I wouldn't understand "climb the stars".

File: QUEST.tsv [hunk 3]
```diff
@@ -1499,155 +1499,155 @@ QUEST_20150323_001498	Okay, that'll be only 1,000 silver coins then.
...
 QUEST_20150323_001603	Nature of the Curse
-QUEST_20150323_001604	Gluttony, Wrath, Sloth, etc. {nl}The nature of all curses that affect mankind is obsession. {nl}It is like the spirit of the dead remaining behind and not going to the Goddess. 
-QUEST_20150323_001605	The difference is that it is the will of the dead that keeps the spirit behind {nl}while curses are forced upon by others onto you.{nl}
-QUEST_20150323_001606	Therefore, if you are a priest learning the teaching of the Goddess, you are not permitted to use it on people unless for the purpose of experimentation.
+QUEST_20150323_001604	$Gluttony, Wrath, Sloth, etc. {nl}The nature of all curses that affect mankind is obsession. {nl}It's like the spirit of the dead staying behind and not going to the Goddess. 
+QUEST_20150323_001605	$The difference is that it is the will of the dead that keeps the spirit behind {nl}while curses are forced upon by others onto you.
+QUEST_20150323_001606	$Naturally, priests learning the teachings of the Goddess are not permitted to use it unless its for experiments.
```
Comment (terrapin47) [inline QUEST.tsv]: its -> it's

**Comment by imcgames** [comment]
Thank you :smile:

**Pull request merged**